AccessControl: Document basic roles changes and provisioning V2 (#48910)
grafana/grafana: docs/sources/enterprise/access-control/about-rbac.md

- Editor
- Viewer

Each basic role is comprised of a number of _permissions_. For example, the viewer basic role contains the following permissions among others:

- `Action: datasources.id:read, Scope: datasources:*`: Enables the viewer to see the ID of a data source.

> - `Action: orgs:read`: Enables the viewer to see his organization details

Reviewer: ```suggestion
- `Action: orgs:read`: Enables the viewer to see their organization details
```

- `Action: annotations:read, Scope: annotations:*`: Enables the viewer to see annotations that other users have added to a dashboard.
- `Action: annotations:create, Scope: annotations:type:dashboard`: Enables the viewer to add annotations to a dashboard.
- `Action: annotations:write, Scope: annotations:type:dashboard`: Enables the viewer to modify annotations of a dashboard.
- `Action: annotations:delete, Scope: annotations:type:dashboard`: Enables the viewer to remove annotations from a dashboard.

> You can use RBAC to modify the permissions associated with any basic role, to modify what viewers, editors, or admins can do. For more information about the permissions associated with each basic role, refer to [Basic role definitions]({{< relref "./rbac-fixed-basic-role-definitions#basic-role-assignments" >}}).

Reviewer: ```suggestion
You can use RBAC to modify the permissions associated with any basic role, which changes what viewers, editors, or admins can do. For more information about the permissions associated with each basic role, refer to [Basic role definitions]({{< relref "./rbac-fixed-basic-role-definitions#basic-role-assignments" >}}).
```

> You can use RBAC to modify the permissions associated with any basic role, which changes what viewers, editors, or admins can do. For more information about the permissions associated with each basic role, refer to [Basic role definitions]({{< relref "./rbac-fixed-basic-role-definitions#basic-role-assignments" >}}).

Reviewer 2: Shall we document that you can't delete the basic role?

> **Note:** You must assign each Grafana user a basic role.

> To interact with the API and view or modify basic roles permissions, refer to the following mapping:

Reviewer: ```suggestion
To interact with the API and view or modify basic roles permissions, refer to the following table that maps basic role names to the associated UID:
```

> To interact with the API and view or modify basic roles permissions, refer to the following table that maps basic role names to the associated UID:

Reviewer 2: This information seems to be included already in "List permissions associated with roles" section under Manage RBAC roles. What do you think about simply referencing it to that page? For example

"To see the UIDs associated with each basic role, refer to Manage RBAC roles"
